try {
    hello();
} catch(err){
    console.log(err)
}



//es10

try {
    anotherFn()
} catch {
    console.log('esto es un error')
}